parsec/core/fs/storage/user_storage.py

```diff
@@ -72,15 +72,15 @@ def get_user_manifest(self):
 
     async def _load_user_manifest(self) -> LocalUserManifest:
         try:
-            await self.manifest_storage.get_manifest(self.user_manifest_id)
+            return await self.manifest_storage.get_manifest(self.user_manifest_id)
```

Comment:
This didn't cause any error ? (and no test cached that ?...)

If it's the case wouldn't it be better to have `-> None` as return type instead ?

Reply:
It can be `-> None` because it's only called once without assignment: 
```
                await self._load_user_manifest()
```